portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/clustering/clusteringframework/ClusteringCache.testcase

```diff
@@ -330,4 +267,89 @@ definition {
 			scriptFile = "groovy-script-portal-cache-get.groovy");
 	}
 
+	@priority = 4
+	test SessionReplicatedIdAndAttributesAreTheSameInBothNodes {
+		property cluster.session.replicated = "true";
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}portlet.session.replicate.enabled=true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		Clustering.deployClusteringModule(nodePort = 8080);
+
+		Clustering.deployClusteringModule(nodePort = 9080);
+
+		ProductMenu.gotoPortlet(
+			category = "Site Builder",
+			portlet = "Pages");
+
+		PagesAdmin.addPage(pageName = "Session Replication Test");
+
+		Navigator.gotoPage(pageName = "Session Replication Test");
+
+		Portlet.addPG(portletName = "ClusterSampleSessionReplication");
+
+		JSONUser.addUser(
+			userEmailAddress = "test1@liferay.com",
+			userFirstName = "Test1",
+			userLastName = "Test1",
+			userScreenName = "Test1");
+
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "test1@liferay.com");
+
+		Open(value1 = "http://www.able.com:9080/web/guest/session-replication-test");
+
+		var guestSessionId = selenium.getText("ClusterSamplePortlet#CURRENT_SESSION_ID");
+
+		User.firstLoginUI(
+			password = PropsUtil.get("default.admin.password"),
+			specificURL = "http://www.able.com:9080",
+			userEmailAddress = "test1@liferay.com");
+
+		Open(value1 = "http://www.able.com:9080/web/guest/session-replication-test");
+
+		AssertElementNotPresent(locator1 = "ClusterSamplePortlet#ALL_SESSION_IDS_CONTAINS_GUEST_SESSION_ID");
```

Comment:
This session was added to make sure that guest session is replaced by logged in session and no longer exist after log in